src/models/Config.ts

```diff
@@ -99,6 +99,7 @@ export const baseConfig = {
         custom: "CUSTOM",
         people: "PEOPLE",
         benefit: "BENEFIT",
+        "reportable sub": "SUB_REPORTABLE",
```

Comment:
These are internal keys to reference, so they should be nice camelCase -> `reportableSub`